src/seml/start.py

```diff
@@ -123,6 +125,15 @@ def get_command_from_exp(
                 f"Starting debug server with IP '{ip_address}' and port '{port}'. "
                 f'Experiment will wait for a debug client to attach.'
             )
+            previous_root = logging.root
+            try:
+                logging.basicConfig(force=True, level=logging.INFO)
+                logging.info(
+                    "If you are using VSCode, you can use the 'Debug Launcher' extension to attach: \n"
+                    f'{_generate_debug_attach_url(ip_address, port)}'
```

Comment:
If the link is too long, using logging.info may not be ideal. Imo, there are two solutions here: use `print`, which directly writes to the system out buffer, or use rich's markdown to define links https://rich.readthedocs.io/en/stable/markup.html#links the first is more robust, the latter looks better :) I guess using rich is more in line with the rest of the code, where we try to minimize print statements.

Reply:
Great idea, I have now switched to using `seml.console` directly.